addons/website_blog/controllers/main.py

```diff
@@ -170,6 +170,14 @@ def tags_list(tag_ids, current_tag):
         response = request.website.render("website_blog.blog_post_short", values)
         return response
 
+    @http.route(['/blog/<model("blog.blog"):blog>/<any("atom","rss"):feed_type>'], type='http', auth="public", website=True)
+    def blog_feed(self, blog, feed_type, **post):
+        limit = int(post.get('limit', 15))
+        cr, uid = request.cr, request.uid
+        base_url = request.registry['ir.config_parameter'].get_param(cr, uid, 'web.base.url')
+        response = request.website.render("website_blog.website_blog_%s" % (feed_type), {'blog': blog, 'base_url': base_url, 'limit': limit})
```

Comment:
Actually, make a `search_read` to get the blog post (with the limit). This will avoid access problem on to read `author_id.name`, since you will receive a name_get ([id, name]). Drop the `email` field from the 2 templates.